CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+Add: include from in log context (#918)
```

Comment:
Do we have some document in this repo describing the log system? In positive case, it should be updated due to ths PR. In negative case, this is NTC.

Reply:
Not sure you mean something like this: https://github.com/telefonicaid/iotagent-node-lib/blob/96e12e12847dda7afd8cf6fb7af243a0ca9970c2/doc/operations.md#logs

Reply:
NTC